src/api/PermissionsApi.js

```diff
@@ -37,17 +37,23 @@ const LOG = new Logger('PermissionsApi');
  * @static
  * @memberof lattice.PermissionsApi
  * @param {UUID[]} aclKey
- * @returns {Promise<Acl>}
+ * @returns {Promise<AclObject>}
  *
  * @example
  * PermissionsApi.getAcl(["ec6865e6-e60e-424b-a071-6a9c1603d735"]);
  */
-function getAcl(aclKey :UUID[]) :Promise<Acl> {
+function getAcl(aclKey :UUID[]) :Promise<AclObject> {
 
   let errorMsg = '';
 
-  if (!isValidUUIDArray(aclKey)) {
-    errorMsg = 'invalid parameter: aclKey must be a non-empty array of valid UUIDs';
+  if (!isNonEmptyArray(aclKey)) {
+    errorMsg = 'invalid parameter: "aclKey" must be a non-empty array';
```

Comment:
how about `if(isEmptyArray(aclKey)` instead of the double negative

Reply:
I agree about the double negative, it's not ideal. Your suggestion means it will only trigger an error if `aclKey` basically equals `[]`, which is not right because `aclKey` could be a string, number, etc. and we want to error in that case as well.

Reply:
Ok, if I understand this correctly `aclKey` should strictly be an array of UUIDs. Shouldn't the check therefore be `!isValidUUIDArray(aclKey)`

Reply:
!isValidUUIDArray(aclKey) is totally fine, but I'm trying to remove it. This is one tiny step towards migrating to lattice-utils for validation.

Reply:
ok, thanks for clearing that up.